data/avgi/avgi ships.txt

```diff
@@ -0,0 +1,2142 @@
+	description "While not as modular as the Koryfi and Avlaki designs, the Melodikos shares the same solar array and structural frame as the Harmonikos. Refits to swap the centrifuge for a set of cargo pods are uncommon due to the work involved, but still possible."
+	description "While its standard set of magnetoplasmadynamic thrusters give it an acceleration generously described as sluggish, the Melodikos' constant thrust flight profile allows it to reach an impressive cruising velocity given enough time."
+
+
+ship "Phononos"
+	plural "Phononos"
+	sprite "ship/avgi phononos"
+	thumbnail "thumbnail/avgi phononos"
+	attributes
+		category "Light Freighter"
+		licenses
+			"Avgi Civilian"
+		"shields" 240
+		"hull" 1500
+		"automaton" 1
+		"mass" 36
+		"drag" .24
+		"heat dissipation" 0.9
+		"fuel capacity" 300
+		"cargo space" 240
+		"outfit space" 72
+		"engine capacity" 24
+		"radar jamming" 6
+		weapon
+			"blast radius" 24
+			"shield damage" 420
+			"hull damage" 360
+			"hit force" 180
+	outfits
+		"Concentrating Solar Array"
+		"Ultracapacitor Cell"
+		
+		"Magnetoplasma Drive"
+		"Hyperdrive"
+
+	engine -9 -19
+		angle 15
+	engine 9 -19
+		angle -19
+	"reverse engine" -9 -19
+		angle -15
+		zoom 0.5
+	"reverse engine" 9 -19
+		angle 15
+		zoom 0.5
+	"steering engine" -9 -19
+		angle 15
+		zoom 0.5
+		right
+	"steering engine" 9 -19
+		angle -15
+		zoom 0.5
+		left
+	"steering engine" -9 -19
+		angle 165
+		zoom 0.5
+		left
+	"steering engine" 9 -19
+		angle 195
+		zoom 0.5
+		right
+
+	leak "leak" 30 60
+	explode "tiny explosion" 12
+
+	description "This retrofit of the Photikon is little more than a set of cargo pods joined to the front of the torpedo bomber, with struts taking the place of the hardpoints and pylons usually used to mount weapons. An expanded internal tank to fuel a hyperdrive has been barely crammed into the ship, at the cost of reduced space for other internal systems and a slightly weaker hull. Because of this, the Phononon is quite vulnerable to Dissonance piracy or even Abberant incursions, and is thus always part of a larger fleet convoy. Even still, as an automated freighter it experiences much higher rates of accidents and ship loss, a fact that has kept crewed vessels like the Avlaki competitive."
+	description "Using the same standard cargo containers as the Avlaki and Harmonikos allows it to carry an impressive amount of cargo, which can be quickly swapped out at the destination for an empty set of shipping containers."
+
+
+ship "Frequentia"
+	sprite "ship/avgi frequentia"
+	thumbnail "thumbnail/avgi frequentia"
+	swizzle 0
+	attributes
+		category "Utility"
+		licenses
+			"Avgi Civilian"
```

Comment:
```suggestion
		licenses
			"Avgi Civilian"
		"cost" 1920000
```
I'm not really sure how to price a tanker, but 20 fuel pods cost about as much as 4-5 KP-6 photovoltaic arrays, so based on that I've suggested a similar price to the Avlaki.